web/pdf_page_view.js

```diff
@@ -588,10 +588,14 @@ class PDFPageView {
   }
 
   toggleEditingMode(isEditing) {
+    // The can be invisible, consequently there's no annotation layer and we
```

Comment:
There seems to be a word missing here.
```suggestion
    // The page can be invisible, consequently there's no annotation layer and we
```